packages/hash/frontend/src/components/auth/signup/OrgInvite.tsx

```diff
@@ -37,7 +37,7 @@ export const OrgInvite: VFC<OrgInviteProps> = ({
   >(createOrgEmailInvitationMutation, {
     onCompleted: (_) => {},
     onError: ({ graphQLErrors }) => {
-      const errorMsg = graphQLErrors?.[0].message;
+      const errorMsg = graphQLErrors[0]!.message;
```

Comment:
Was it intentional to remove the optional access on `graphQLErrors`?

Reply:
Yep, `?.` was not necessary because `graphQLErrors` cannot be `undefined`

<img width="614" alt="Screenshot 2022-03-14 at 11 48 30" src="https://user-images.githubusercontent.com/608862/158166147-96353e87-5e1e-45a2-8cb3-51acad1b68f7.png">
<img width="563" alt="Screenshot 2022-03-14 at 11 48 45" src="https://user-images.githubusercontent.com/608862/158166151-05465e0f-dbb6-4200-9805-f7b769ca80b0.png">